llvm/lib/Target/ARM/ARMFrameLowering.h

```diff
@@ -59,6 +59,7 @@ class ARMFrameLowering : public TargetFrameLowering {
   void determineCalleeSaves(MachineFunction &MF, BitVector &SavedRegs,
                             RegScavenger *RS) const override;
 
+  void updateLRRestored(MachineFunction &MF) const;
```

Comment:
Does this function still use anything from ARMFrameLowering? If not, could it be made static? Would also be good to document the new dedicated function

Reply:
It can't be static because it's used from `ARMLoadStoreOptimizer`, but I'll add a doc comment for it.